spec/views/forms/make_live/confirmation.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe "forms/make_live/confirmation.html.erb" do
+  around do |example|
+    ClimateControl.modify RUNNER_BASE: "runner-host" do
+      example.run
+    end
+  end
+
+  before do
+    assign(:form, OpenStruct.new(id: 1, name: "Form 1", form_slug: "form-1", status: "draft", pages: nil))
```

Comment:
why does it need `status` and `pages`?

Reply:
Good catch. I copied this line from the tests for `show.html`, except replacing `pages:` with `nil`, but you're right, it shouldn't need to be there.

It is surprising though that the view would work with `status: draft`, but I guess the routing logic is what takes care of that?